src/components/Incidents/IncidentsPage.jsx

```diff
@@ -10,67 +10,62 @@ import { useDispatch } from 'react-redux';
 import * as _ from 'lodash-es';
 import { processIncidentsTimestamps } from './utils';
 
-const minSamples = 10;
-const maxSamples = 300;
-const minStep = 5 * 1000;
-const getMaxSamplesForSpan = (span) => _.clamp(Math.round(span / minStep), minSamples, maxSamples);
-const spans = ['1d', '3d', '7d', '15d'];
+const twoWeeksDateRanges = [
+  { endTime: 1723625094404, duration: 86400000 },
+  { endTime: 1723711494404, duration: 86400000 },
+  { endTime: 1723797894404, duration: 86400000 },
+  { endTime: 1723884294404, duration: 86400000 },
+  { endTime: 1723970694404, duration: 86400000 },
+  { endTime: 1724057094404, duration: 86400000 },
+  { endTime: 1724143494404, duration: 86400000 },
+  { endTime: 1724229894404, duration: 86400000 },
+  { endTime: 1724316294404, duration: 86400000 },
+  { endTime: 1724402694404, duration: 86400000 },
+  { endTime: 1724489094404, duration: 86400000 },
+  { endTime: 1724575494404, duration: 86400000 },
+  { endTime: 1724661894404, duration: 86400000 },
+  { endTime: 1724748294404, duration: 86400000 },
+];
 
 const IncidentsPage = ({
   customDataSource,
   defaultTimespan = parsePrometheusDuration('7d'),
-  defaultSamples,
   namespace,
   timespan,
-  fixedXDomain,
 }) => {
   const [startingDate, setStartingDate] = React.useState('');
   const [endDate, setEndDate] = React.useState('');
   const [incidentsPageData, setIncidentsPageData] = React.useState([]);
-  const defaultSpanText = spans.find((s) => parsePrometheusDuration(s) >= defaultTimespan);
-  const [span, setSpan] = React.useState(timespan || parsePrometheusDuration(defaultSpanText));
   //raw data
   const [data, setData] = React.useState([]);
   //data that is mapped and changed from timestamps to a format HH/DD/MM/YY
   const [processedData, setProcessedData] = React.useState([]);
   //will be used to define the Xdomain of chart
-  const [xDomain, setXDomain] = React.useState();
-  const maxSamplesForSpan = defaultSamples || getMaxSamplesForSpan(span);
-
-  const [samples, setSamples] = React.useState(maxSamplesForSpan);
-  const getXDomain = (endTime, span) => [endTime - span, endTime];
-  React.useEffect(() => {
-    setXDomain(getXDomain(Date.now(), span));
-  }, [span, fixedXDomain]);
-  const endTime = xDomain?.[1];
-
-  React.useEffect(() => {
-    if (timespan) {
-      setSpan(timespan);
-      setSamples(defaultSamples || getMaxSamplesForSpan(timespan));
-    }
-  }, [defaultSamples, timespan]);
+  const [xDomain, setXDomain] = React.useState(twoWeeksDateRanges);
```

Comment:
Is there a reason other than using test data that is consistent that the old "find 2 weeks of dates" was removed? If there is then don't worry about this comment, but if it's just going to have to come back in a future PR then we should just leave it even just commented out or something.

Reply:
This is a temporary thing and will be changed in the next PR